EventFilter/L1TRawToDigi/src/OmtfLinkMappingCsc.cc

```diff
@@ -0,0 +1,85 @@
+#include "EventFilter/L1TRawToDigi/interface/OmtfLinkMappingCsc.h"
+
+namespace omtf {
+
+MapEleIndex2CscDet mapEleIndex2CscDet() {
+
+  MapEleIndex2CscDet result;
+  for (unsigned int fed=1380; fed<=1381; fed++) {
+    //Endcap label. 1=forward (+Z); 2=backward (-Z)
+    unsigned int endcap = (fed==1380) ? 2 : 1;
+    for (unsigned int amc=1;    amc<=6; amc++) {
+      for (unsigned int link=0; link <=34; link++) {
```

Comment:
this loop makes no sense. make a look up tables for stat,ring,cham. and then you don't need the complex case statement

Reply:
I agree. 
I would leave this for developer to make this improvement in a separate PR, if you are OK with  it.